docs/usage-as-end-user.md

```diff
@@ -61,6 +61,27 @@ apiVersion: packet.provider.extensions.gardener.cloud/v1alpha1
 kind: ControlPlaneConfig
 ```
 
+## `WorkerConfig`
+
+The Packet extension supports specifying IDs for reserved devices that should be used for the machines of a specific worker pool.
+
+An example `WorkerConfig` for the Packet extension looks as follows:
+
+```yaml
+apiVersion: packet.provider.extensions.gardener.cloud/v1alpha1
+kind: WorkerConfig
+reservationIDs:
+- my-reserved-device-1
+- my-reserved-device-2
+reservedDevicesOnly: false
+```
+
+The `.reservationIDs[]` list contains the list of IDs of the reserved devices.
+The `.reservedDevicesOnly` field indicates whether only reserved devices should be used (based on the list of reservation IDs)
+when new machines are created.
+If it's set to `false` and the list of reservation IDs is exhausted then the next available device (unreserved) will be
```

Comment:
I would change the language here to:

```
The `.reservedDevicesOnly` field indicates whether only reserved devices from the provided list of reservation IDs should be used when new machines are created. It always will attempt to create a device from one of the reservation IDs. If none is available, the behaviour depends on the setting:

* `true`: return an error
* `false`: request a regular on-demand device
```

Also, what is the default?

Reply:
done in d514ede7cb983e27237ce6301f51ec1004785630